src/frontend/src/catalog/database_catalog.rs

```diff
@@ -42,6 +45,18 @@ impl DatabaseCatalog {
         self.schema_name_by_id.try_insert(id, name).unwrap();
     }
 
+    // For backward compatibility, since we didn't reject some operations on `rw_catalog` and
+    // `information_schema`: 1.drop schema, 2. create relations in them, here we have to replace
+    // back these schemas in frontend.
+    fn replace_sys_schema_id_tmp(&mut self, proto: &PbSchema) {
```

Comment:
Workaround in frontend for compatibility with some operations that were not banned in previous versions.